routes.py

```diff
@@ -1,12 +1,13 @@
 from flet_route import path
-from views import indexView, aboutAppView, aboutAppCity, museumInfoView, settings, allEventsViews
+from views import indexView, aboutAppView, aboutAppCity, museumInfoView, settings, allEventsViews,eventInfo
 
 app_routes = [
     path(url="/", clear=True, view=indexView),
     path(url="/mus/:id", clear=False, view=museumInfoView),
     path(url='/about_app/', clear=False, view=aboutAppView),
     path(url='/about_city/', clear=False, view=aboutAppCity),
     path(url='/settings/', clear=False, view=settings),
-    path(url='/events/', clear=False, view=allEventsViews),
+    path(url='/event/', clear=False, view=allEventsViews),
```

Comment:
Если по этому пути будет находится список событий то лучше использовать множественное число:
```suggestion
path(url='/events/', clear=False, view=allEventsView),
```
Плюс вьюху сделай в единственном числе.